### DISCUSSION TITLE
Add a enviroment variable to disable confirmation mails for sign ups, organisaton invitations etc

### DISCUSSION THREAD

**Discussion by Seb-AM** [dani-garcia/vaultwarden#2094] [Ideas]
I am hosting a VW instance without activated sign ups, nor invites. All users  of my instance are curated via a LDAP Group.

For me the "E-Mail spam" of the several confirmation mechanics in Bitwarden/Vaultwarden are no use to me. 
I am sure that is the case for other too.

So I would like to see one or more enviroment variable where I can toggle the confirmation emails for new members of the plattform and organisations.

**Comment by kl4ppdrachen** [comment]
I would recommend this, too. We use it on a local network without an internet connection.
